db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
Task requires that each player should choose a race to play, such as Elf, Dwarf, Human, or Ork.
You have to create variable RACES_NAME inside of the Race class and add argument choices to name field.